### PR TITLE
baylands error message

### PR DESCRIPTION
**Pull request by frede791**

### CODE REVIEW

File: en/sim_gazebo_gz/README.md [hunk 1]
```diff
@@ -105,6 +105,10 @@ However you must not _explicitly_ specify `_default` on the model as this will p
 In other words, use `make px4_sitl gz_x500` instead of `make px4_sitl gz_x500_default` for the default.
 :::
 
+:::warning
+Baylands has a lot of meshes and will throw the following error: `[Err] [SDFFeatures.cc:843] The geometry element of collision [collision] couldn't be created`. This is expected and does not break the functionality of Gazebo. In the newest release, Gazebo Harmonic, this has been downgraded to a [warning](https://github.com/gazebosim/gz-physics/pull/531). As such, it can be ignored when using baylands. If you are particularly worried about it, you can also [install gz-garden from source](https://gazebosim.org/docs/garden/install_ubuntu_src) as the warning has been backported as well.
```
Comment (hamishwillee) [inline en/sim_gazebo_gz/README.md:109]:
```suggestion
Baylands has a lot of meshes and will throw the following error: [Err] [SDFFeatures.cc:843] The geometry element of collision [collision] couldn't be created`.
This is expected: it does not break the functionality of Gazebo and can hence be ignored.

The error has been downgraded to a warning in Gazebo Harmonic: [gz-physics/pull/531](https://github.com/gazebosim/gz-physics/pull/531).
If you are particularly worried about the warning you can squash it by [installing gz-garden from source](https://gazebosim.org/docs/garden/install_ubuntu_src) (the warning has been backported).
```

Comment (frede791) [reply en/sim_gazebo_gz/README.md:109]: @hamishwillee Sounds better than what I wrote :)

File: en/sim_gazebo_gz/README.md [hunk 2]
```diff
@@ -105,6 +105,10 @@ However you must not _explicitly_ specify `_default` on the model as this will p
 In other words, use `make px4_sitl gz_x500` instead of `make px4_sitl gz_x500_default` for the default.
 :::
 
+:::warning
```
Comment (hamishwillee) [inline en/sim_gazebo_gz/README.md:108]: I think a note. It's important, but not "scary"

```suggestion
:::note
```

File: en/sim_gazebo_gz/README.md (hunk 1 above)
Comment (hamishwillee) [reply en/sim_gazebo_gz/README.md]: Yes, except cursed Github lost it. Trying again.

**Review by hamishwillee — APPROVED** [review]
Thanks @frede791 - thanks for posting. I mean that - so often there are warnings and it isn't clear if they are known and everyone is ignoring them, or they are new and should be reported.

File: en/sim_gazebo_gz/README.md (hunk 2 above)
Comment (frede791) [reply en/sim_gazebo_gz/README.md]: @hamishwillee I don't know. If I see a bright red error message, I would read it and ctrl + C out of Gazebo. I think it's important to highlight that this error is expected.

**Comment by hamishwillee** [comment]
> @hamishwillee I don't know. If I see a bright red error message, I would read it and ctrl + C out of Gazebo. I think it's important to highlight that this error is expected.

Sorry missed this. I agree it is important. How about we see whether it is prominent enough after it renders.

**Pull request merged**